src/bundle/ui-dev/src/modules/universal-discovery/components/grid-view/grid.view.item.js

```diff
@@ -73,7 +73,7 @@ const GridViewItem = ({ location, version }) => {
                     contentTypeIconPath={contentTypesMap[location.ContentInfo.Content.ContentType._href].thumbnail}
                 />
             </div>
-            <div className="c-grid-item__name">{location.ContentInfo.Content.TranslatedName}</div>
+            <div className="c-grid-item__name"><span className="c-grid-item__name-text">{location.ContentInfo.Content.TranslatedName}</span></div>
```

Comment:
Is this additional span needed? In the previous design, it looked similar and the span was not required.